src/actions/actionDistribute.tsx

```diff
@@ -49,7 +49,7 @@ export const distributeHorizontally = register({
     };
   },
   keyTest: (event) => {
-    return event.altKey && event.key === KEYS.H_KEY;
+    return event.altKey && event.code === KEYS.H_KEY;
```

Comment:
I'm getting lost in what is a `key` and what is a `code`, esp since `H_KEY` is called a KEY. Let's split the constants into keys and codes?

Reply:
I'll update the pr